TelCo.ColorCoder/ColorList.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// The 25-pair color code, originally known as even-count color code, 
+    /// is a color code used to identify individual conductors in twisted-pair 
+    /// wiring for telecommunications.
+    /// This class provides the color coding and 
+    /// mapping of pair number to color and color to pair number.
+    /// </summary>
+    public class ColorList
+    {
+        /// <summary>
+        /// Array of Major colors
+        /// </summary>
+        public static Color[] colorMapMajor;
```

Comment:
member variables should not be public. expose through only get property accessor